srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -59,61 +63,70 @@ public RpcVistaPatientDao(
         fDuz = duz;
     }
     
+    /**
+     * {@inheritDoc}
+     * <p>This method will eager load all available information about the patient including vitals,
+     * basic information, and lab results. Eager loading is done to sacrifice efficiency for
+     * simplicity. The same thing is done in VistA CPRS</p>
```

Comment:
Good. I would clarify about the CPRS comment, though, and say "the number of eager remote procedure calls is comparable to CPRS" or something like that.